WNPRC_EHR/resources/schemas/dbscripts/postgresql/wnprc-21.002-21.003.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE wnprc.watermonitoring_access
+(
+    rowid                   serial                    NOT NULL,
+    date                    TIMESTAMP,
+    alloweduser             userid,
+    project                 integer,
+    createdBy               userid,
```

Comment:
I've been adding all of the following columns to the tables I create for slightly better tracking of how/when rows were inserted/modified. Not sure if that would be useful here or not, just a thought.

    createdby       userid,
    created         TIMESTAMP,
    modifiedby      userid,
    modified        TIMESTAMP,

Reply:
Followed your advise, I added those columns.